dm/dm/ctl/master/config.go

```diff
@@ -79,6 +80,25 @@ func newConfigTaskCmd() *cobra.Command {
 	return cmd
 }
 
+func newConfigTaskTemplateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "template [task-name]",
```

Comment:
too hard to give this cmd a right name 😂  task-config? template? OpenAPITask(this is wrong)?

the usage scenarios is  user create task config in WebUI, but want to dump this config to file and operate this task config with dmctl, neew suggestions here